pkg/querier/querier.go

```diff
@@ -158,7 +158,20 @@ func (q querier) LabelValues(name string) ([]string, error) {
 }
 
 func (q querier) LabelNames() ([]string, error) {
-	return nil, nil
+	return q.distributor.LabelNames(q.ctx)
+
+	// TODO(codesome): Would the above make sense or the below?
```

Comment:
I have 2 solutions for this function. Would `q.distributor.LabelNames(q.ctx)` be fine or the commented solution is the way to do?

Reply:
For now there is no way to implement it for the chunkstore, so what you have above should be fine.